<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageIntersite</class>
 <widget class="QDialog" name="ManageIntersite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>292</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>442</width>
    <height>292</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>442</width>
    <height>292</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="staName">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="intersiteTable">
     <property name="minimumSize">
      <size>
       <width>420</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>200</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="addIntersite">
       <property name="minimumSize">
        <size>
         <width>92</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>92</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>添加连接</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icon/icon/add.png</normaloff>:/icon/icon/add.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
